main.nf

```diff
@@ -394,6 +394,81 @@ process sentieon_qc {
 	"""
 }
 
+process sentieon_mitochondrial_qc {
+
+    // Fetch mitochondrial coverage statistics
+    // Calculate mean_coverage and pct_above_500x
+    
+    cpus 52
+    memory '30 GB'
+	tag "$id"
+	cache 'deep'
+	time '2h'
+	scratch true
+	stageInMode 'copy'
+	stageOutMode 'copy'
+	container = "/fs1/resources/containers/sentieon_202112.sif"
+
+	when:
+		params.antype == "wgs"
+    
+	input:
+        set group, id, file(bam), file(bai) from qc_mito_bam.groupTuple()
+
+	output:
+    	group, id, file("${id}_mito_coverage.tsv") into <<what?>>
```

Comment:
I'm only saving the summary file with the relevant QC data and (I think) throwing away everything else.